src/dmd/traits.d

```diff
@@ -1810,6 +1811,41 @@ Lnext:
         }
         return r.expressionSemantic(sc);
     }
+    if (e.ident == Id.getLocation)
+    {
+        //Symbol as far as we're concered here can be:
+        // a type,
+        // a dsymbol,
+        // an expression of type DsymbolExp
+
+        if (dim != 1)
+            return dimError(1);
+        auto arg0 = (*e.args)[0];
+        Dsymbol s;
```

Comment:
Rather use:

```d
Dsymbol s = getDsymbolWithoutExpCtx(arg0);
```

instead of extracting the symbol and possibly create problems.

Reply:
Thanks I didn't know about that.